packages/next/client/image.tsx

```diff
@@ -45,6 +45,11 @@ const VALID_LAYOUT_VALUES = [
 ] as const
 type LayoutValue = typeof VALID_LAYOUT_VALUES[number]
 
+enum PlaceholderValue {
+  BLURRY = 'blurry',
+  EMPTY = 'empty',
+}
```

Comment:
How about `type PlaceholderValue = 'blurry' | 'empty'` so that there is no runtime variable?

Reply:
Using enums allows more specific type definitions using the various enum values themselves instead of just the type (See line 82-86). I see that this is achieved currently for other types in this file by just rewriting the string literals, but I much prefer to use declared constants (in this case the enum values) whenever possible (to avoid errors from misspelling, etc).

Reply:
There shouldn't be any errors from misspelling because TS will error if the string doesn't match the union. I think you should follow the pattern that Next.js uses instead of your personal preference.

Reply:
Happy to switch to follow Next.js patterns. I just wanted to clarify that I wasn't trying to impart my personal preference upon the code "just because", but rather those preferences were born from previous learnings of good practices. It's clear that there's still details for me to learn that may be TS specific, and I'm happy to revise and improve my knowledge of those practices as I learn more.